StRoot/StETofUtil/etofHelperFunctions.cxx

```diff
@@ -0,0 +1,151 @@
+#include <cmath>
+
+#include "StETofUtil/etofHelperFunctions.h"
+
+#include "SystemOfUnits.h"
+#include "PhysicalConstants.h"
+
+#include "StMessMgr.h"
+#include "StBTofHeader.h"
+
+
+// *******************************************************************************************************
+//
+double
+calculateVpdTstart( const StBTofHeader* btofHeader, const double& pVtxZ, const double& bTofClockRange )
```

Comment:
How is this function related to the ETOF clock jump correction? I don't see how anything in StRoot/StETofUtil/etofHelperFunctions.cxx is used either StETofMatchMaker or StETofHitMaker but please correct me if I am wrong.

Reply:
The tofPathLength function is used in the StEtofHitmaker for the doublejump correection ()

Reply:
Already defined in StTofUtil
https://github.com/star-bnl/star-sw/blob/1440089feb9da2b7c27e00bd2f7b776c9867ff9d/StRoot/StTofUtil/tofPathLength.cc#L98